docs/assets/css/custom-molfeat.css

```diff
@@ -43,6 +43,10 @@
   color: rgb(255, 255, 255);
 }
 
+.md-search__input {
```

Comment:
```suggestion
.md-search .md-search__input {
```

Reply:
Thanks @cwognum, it's because of the position, should be fixed by moving it at the right place.